torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -32,7 +32,9 @@ struct RpcBackendOptions {
 
   RpcBackendOptions(float rpcTimeoutSeconds, std::string initMethod)
       : rpcTimeoutSeconds(rpcTimeoutSeconds),
-        initMethod(std::move(initMethod)) {}
+        initMethod(std::move(initMethod)) {
+    TORCH_CHECK(rpcTimeoutSeconds > 0, "RPC Timeout must be greater than 0");
```

Comment:
Do we accept 0 timeout as infinite timeout? The args doc in `rpc_sync` says:

```
        timeout (float, optional): timeout in seconds to use for this RPC. If
                                   the RPC does not complete in this amount of
                                   time, an exception indicating it has
                                   timed out will be raised. A value of 0
                                   indicates an infinite timeout, i.e. a timeout
                                   error will never be raised. If not provided,
                                   the default value set during initialization
                                   or with `_set_rpc_timeout` is used.
```

Reply:
Yes, looks like we should support 0 - will update